sdk/RELEASE.md

```diff
@@ -3,8 +3,21 @@
 ## Features
 
 ## Breaking changes
+
+## Deprecations
+
+## Bug fixes and other changes
+
+## Documentation updates
+
+# 2.4.0
```

Comment:
https://github.com/kubeflow/pipelines/pull/10094 should be part of this release?